tools/kmodules/build_module

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eu
+
+MODULE=$(dirname $0)/$1
```

Comment:
I'm thinking, we could pass the path to the module source as a parameter of this script, and provide a way for users to override it (by specifying something `target.conf` for example). We might need several versions of this module in case the in-kernel ABI changes (because of upstream progress, or because of device-specific changes) and keep backward compatibility.

Reply:
Okay. How about default to our known location unless the user specifically wants to override?